### ISSUE TITLE
Require pip option not working

### ISSUE THREAD

**Issue by lobsterdore** [neillturner/kitchen-ansible#185]
Specifying require_pip in the provisioner options has no effect at all, I think this is because the option is missing from config.rb.

Steps to reproduce:

```YAML
# .kitchen.yml
provisioner:
  require_ansible_source: false
  require_pip: true
```

**Expected result:**

Ansible is installed

**Actual result:**

Ansible installation is skipped completely.

**Comment by neillturner** [comment]
i've fixed and will release new version

**Comment by lobsterdore** [comment]
Amazing, thanks very much.

**Comment by neillturner** [comment]
fixed in 0.44.3
